projects/tox4j/src/main/java/im/tox/tox4j/impl/jni/AutoGenerated.java

```diff
@@ -0,0 +1,15 @@
+package im.tox.tox4j.impl.jni;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Mark methods that are generated by JniMethodImpls.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AutoGenerated {
+  /**
+   * Override the C function name this native method should call.
```

Comment:
Added more information. Please take another look and tell me if it requires more or different notes.